main.go

```diff
@@ -683,6 +687,8 @@ func parseFlags() (config, error) {
 		"The address on which the internal server listens.")
 	flag.StringVar(&cfg.server.healthcheckURL, "web.healthchecks.url", "http://localhost:8080",
 		"The URL against which to run healthchecks.")
+	flag.StringVar(&cfg.server.prefixHeader, "web.prefix-header", "X-Forwarded-Prefix",
```

Comment:
It's a good idea to make this configurable, however maybe it's a bit of an overkill. We can simplify implementation by sticking to `X-Forwarded-Prefix` and leave the responsibility of modifying it to an external proxy if needed.